HIP/hip-904.md

```diff
@@ -848,9 +848,6 @@ was transferred.
 message TokenRejectTransactionBody {
     /**
      * A list of one or more token rejections.<br/>
-     * Each token rejection SHALL dissociate the requesting account from a single
```

Comment:
You can be explicit and not that after rejection a user is still associated with the token and can take the measure to submit a TokenDissociate as needed.